apps/api/src/domain/models/bibliographic-reference/entities/BookBibliographicReferenceData.ts

```diff
@@ -1,14 +1,13 @@
 import {
-    IsInt,
     IsISBN,
     IsNonEmptyArray,
     IsOptional,
-    IsPositive,
+    IsPositiveInteger,
     IsStringWithNonzeroLength,
     IsUrl,
```

Comment:
Will there be IsValidUrl for validating url's for media items ?

Reply:
Yes. We should be using this in the song model as well.